src/Reveal.Sdk.Dom.Tests/Visualizations/OHLCVisualizationFixture.cs

```diff
@@ -0,0 +1,275 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Reveal.Sdk.Dom.Core.Serialization;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Filters;
+using Reveal.Sdk.Dom.Visualizations;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations;
+
+public class OHLCVisualizationFixture
+{
+    [Fact]
+    public void Constructor_InitializesDefaultValues_WhenInstanceIsCreated()
+    {
+        // Act
+        var ohlcVisualization = new OHLCVisualization();
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
+        Assert.Equal(ChartType.OHLC, ohlcVisualization.ChartType);
+        Assert.Null(ohlcVisualization.Title);
+        Assert.Null(ohlcVisualization.DataDefinition);
+    }
+
+    [Fact]
+    public void Constructor_InitializesOHLCVisualizationWithDataSource_WhenDataSourceItemIsProvided()
+    {
+        // Arrange
+        var dataSourceItem = new DataSourceItem { HasTabularData = true };
+
+        // Act
+        var ohlcVisualization = new OHLCVisualization(dataSourceItem);
+
+        // Assert
+        Assert.NotNull(ohlcVisualization);
+        Assert.Equal(ChartType.OHLC, ohlcVisualization.ChartType);
+        Assert.Equal(dataSourceItem, ohlcVisualization.DataDefinition.DataSourceItem);
+        Assert.Null(ohlcVisualization.Title);
```

Comment:
Need assertion for DataDefinition, as this constructor has calls to `InitializeDataDefinition` method

Reply:
Need Assertions for `UpdateDataSourceItem` method